dawn.json

```diff
@@ -1824,7 +1824,7 @@
         "members": [
             {"name": "label", "type": "char", "annotation": "const*", "length": "strlen", "optional": true},
             {"name": "format", "type": "texture format", "default": "undefined"},
-            {"name": "dimension", "type": "texture view dimension", "default": "undefined"},
+            {"name": "dimension", "type": "texture view dimension", "default": "2D"},
```

Comment:
I think we should left it as undefined https://bugs.chromium.org/p/dawn/issues/detail?id=329